core/tests/Test.Microsoft.Identity.LabInfrastructure/LabServiceApi.cs

```diff
@@ -71,24 +71,28 @@ private LabResponse GetLabResponseFromAPI(UserQueryParameters query)
 
             queryDict.Add("external", query.IsExternalUser != null && (bool)(query.IsExternalUser) ? "true" : "false");
 
+            if (query.IsUserType == UserType.B2C)
+            {
+                queryDict.Add("b2cProvider", "local");
```

Comment:
How are we going to support non-local accounts? I can several ways: 

enum UserType { B2C_Local, B2C_Google etc.} 

Or 2 enums?

Reply:
The lab supports Google, Amazon, FB, Twitter, MSA accounts. I just started w/local accounts because we seem to have the most issues relating to them.

---
In reply to: [232940688](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r232940688) [](ancestors = 232940688)